src/miner.cpp

```
@@ -289,6 +289,40 @@ std::unique_ptr<CBlockTemplate> BlockAssembler::CreateNewBlock(const CScript& sc
+           spark_outputs.push_back(mintedCoinData);
+           indexes.push_back(i);
+       }
+    }
+
+    if (!spark_outputs.empty()) {
+        CDataStream serialContextStream(SER_NETWORK, PROTOCOL_VERSION);
+        serialContextStream << pindexPrev->GetBlockHash();
+        std::vector<CRecipient> recipients = CSparkWallet::CreateSparkMintRecipients(spark_outputs, std::vector<unsigned char>(serialContextStream.begin(), serialContextStream.end()), true);
+
+        size_t i = 0;
+        for (size_t i = 0; i < recipients.size(); ++i) {
```

cppcheck (2.10-2)

[performance] Prefer prefix ++/-- operators for non-primitive types.

(postfixOperator)